tools/db_crashtest.py

```diff
@@ -238,6 +238,9 @@ def finalize_and_sanitize(src_params):
     if dest_params["index_type"] == 1 and \
         dest_params.get("prefix_size", 7) == -1:
         dest_params["index_type"] = 0       
+    # kHashSearch is incompatible with index_block_restart_interval > 1
```

Comment:
If it is temporary, comment it here saying it's a temporary and should be removed if option sanitization is fixed.

Reply:
It is not temporary. As the PR description says it would avoid confusion.

Reply:
My concern is that, here we exposed a bug. But before fixing the bug, we are masking the error by modifying the test. We would lose the opportunity of validating the future fix.